resources/language/resource.language.ro_ro/strings.po

```diff
@@ -0,0 +1,947 @@
+
+msgctxt "#30211"
+msgid "Another device handles the auto-updates of shared-library"
+msgstr "Alt dispozitiv gestionează actualizările automate ale librăriei partajate"
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of shared-library"
+msgstr "Nu există dispozitiv care să gestioneze actualizările automate ale librăriei partajate"
+
+msgctxt "#30213"
+msgid "InputStream Helper settings..."
+msgstr "Setări add-on InputStream Helper"
+
+msgctxt "#30214"
+msgid "Force the update of my list"
+msgstr "Forțează actualizarea listei mele"
+
+msgctxt "#30215"
+msgid "Color of the titles included in My list"
+msgstr "Culoarea titlurilor incluse în 'Lista mea'"
+
+msgctxt "#30216"
+msgid "Show ESN in use"
+msgstr "Arată ESN utilizat"
+
+msgctxt "#30217"
+msgid "Reset ESN"
+msgstr "Resetează ESN"
+
+msgctxt "#30218"
+msgid "Are you sure you want to reset the ESN?\r\nIf you confirm, wait until you receive the login message."
+msgstr "Sunteți sigur că doriți să resetați ESN?\r\nDacă da, așteptați până când primiți mesajul de autentificare."
+
+msgctxt "#30219"
+msgid "Disable notification for synchronization completed"
+msgstr "Dezactivează notificarea pentru completarea sincronizării"
+
+msgctxt "#30220"
+msgid "The Kodi library has been updated"
+msgstr "Librăria Kodi a fost actualizată"
+
+msgctxt "#30221"
+msgid "Owner account"
+msgstr "Cont principal"
+
+msgctxt "#30222"
+msgid "Kid account"
+msgstr "Cont copil"
+
+msgctxt "#30223"
+msgid "Synchronization now uses \"My List\" of profile {}"
+msgstr "Sincronizarea folosește acum 'Lista mea' aferent[ profilului {}"
+
+msgctxt "#30224"
+msgid "Auto update mode"
+msgstr "Mod auto actualizare"
+
+msgctxt "#30225"
+msgid "Manual"
+msgstr "Manual"
+
+msgctxt "#30226"
+msgid "Scheduled"
+msgstr "Programat"
+
+msgctxt "#30227"
+msgid "Synchronize Kodi library with My List"
+msgstr "Sincronizează librăria Kodi cu 'Lista mea'"
+
+msgctxt "#30228"
+msgid "Use \"My list\" of the current profile"
+msgstr "Folosește 'Lista mea' pentru profilul curent"
+
+msgctxt "#30229"
+msgid "Show the name of the selected profile"
+msgstr "Arată numele profilului selectat"
+
+msgctxt "#30230"
+msgid "Check for updates now"
+msgstr "Verifică acum dacă există actualizări"
+
+msgctxt "#30231"
+msgid "Do you want to check update now?"
+msgstr "Doriți să verificați acum dacă există actualizări?"
+
+msgctxt "#30232"
+msgid "Restrict by Maturity Level"
+msgstr "Restricționează după nivelul de maturitate"
+
+msgctxt "#30233"
+msgid "Content for {} will require the PIN to start playback."
+msgstr "Conținutul pentru {} va avea nevoie de cod pentru a porni redarea."
+
+msgctxt "#30234"
+msgid "Install Up Next add-on"
+msgstr "Instalaează add-on-ul Up Next"
+
+msgctxt "#30235"
+msgid "[WIP] Synchronize the watched status of the videos with Netflix"
+msgstr "[Muncă în desfășurare] Sincronizează statutul conținutului urmărit cu Netflix"
```

Comment:
isn't there an abbreviation like WIP?

Reply:
There is not any in Romanian. I will shorten it.